CMakeLists.txt

```diff
@@ -204,6 +204,13 @@ endif()
 
 find_package(PNG REQUIRED)
 
+option(USE_SYSTEM_FMT "Use preinstalled fmt, must be 8.0.0 or newer" OFF)
+if(USE_SYSTEM_FMT)
+  find_package(fmt 8.0.0 REQUIRED)
+else()
+  add_subdirectory(external/fmt EXCLUDE_FROM_ALL)
+endif()
```

Comment:
Would it be possible (and if yes, would it be a good idea) to always `find_package(fmt 8.0.0)` without making it required, and then include the subdirectory based on whether or not the version was found?

Reply:
Should be an easy enough change if desired, though I can of like sticking with one library version than hoping that whatever gets shipped with a random distribution actually works.